manifests/m/Microsoft/VCLibs/Desktop/14/14.0.30704.0/Microsoft.VCLibs.Desktop.14.installer.yaml

```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.VCLibs.Desktop.14
+PackageVersion: 14.0.30704.0
+Platform:
+- Windows.Universal
+MinimumOSVersion: 10.0.10042.0
+InstallerType: msix
+PackageFamilyName: Microsoft.VCLibs.140.00.UWPDesktop_8wekyb3d8bbwe
+Installers:
+- Architecture: x64
+  InstallerUrl: https://aka.ms/Microsoft.VCLibs.x64.14.00.Desktop.appx
+  InstallerSha256: 9BFDE6CFCC530EF073AB4BC9C4817575F63BE1251DD75AAA58CB89299697A569
+  SignatureSha256: 9D34C5CF8B3951F5E3471206C2DD30EA34B1D7B80ABF9014D21F3109DB87C397
+- Architecture: x86
+  InstallerUrl: https://aka.ms/Microsoft.VCLibs.x86.14.00.Desktop.appx
+  InstallerSha256: 3195DB914BEA1534EE73582CD483C548A929AED2799D305B3BBF7411BA7A6C7D
+  SignatureSha256: D72F5D6E0AF1A1914CD1EDB89A34BCEBD838F2F4395BDD89FE31BF636DCDA351
+- Architecture: arm
+  InstallerUrl: https://aka.ms/Microsoft.VCLibs.arm.14.00.Desktop.appx

```

Comment:
bit worried - this link can be changed on the backend to point to any version at all, and that can happen without this manifest being updated. That will cause widespread install failures.

Non-versioned links are probably not ideal, right?

Reply:
yeah, thanks for catching my rookie mistake